com.carpeludum.KegaFusion.json

```diff
@@ -0,0 +1,86 @@
+{
+    "app-id" : "com.carpeludum.KegaFusion",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "18.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "Fusion",
+    "tags": ["proprietary"],
+    "finish-args" : [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Play sounds */
+        "--socket=pulseaudio",
+        /* Joystick and GPU access */
+        "--device=all",
+        /* Get access to the files */
+        "--filesystem=home:ro",
+        "--filesystem=xdg-run/gvfs:ro"
+    ],
+    "modules" : [
+        "shared-modules/glu/glu-9.json",
+        "shared-modules/gtk2/gtk2.json",
+        {
+            "name" : "kega-fusion",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "make install",
+                "install -D fusion.sh /app/bin/Fusion",
+                "install -D Fusion /app/lib/KegaFusion/Fusion",
+                "install -D Fusion.ini /app/lib/KegaFusion/Fusion.ini",
+                "install -d /app/lib/KegaFusion/Plugins/",
+                "cp *.rpi /app/lib/KegaFusion/Plugins/",
+                "install -d /app/share/doc/KegaFusion",
+                "cp *.txt /app/share/doc/KegaFusion",
+                "install -Dm644 com.carpeludum.KegaFusion.png /app/share/icons/com.carpeludum.KegaFusion.png",
+                "install -Dm644 -t /app/share/applications/ com.carpeludum.KegaFusion.desktop",
+                "install -Dm644 -t /app/share/metainfo/ com.carpeludum.KegaFusion.metainfo.xml"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://www.carpeludum.com/download/Fusion363x.tar.gz",
```

Comment:
Does it's licence allow redistribution? If not (likely so), Fusion should be downloaded at install-time via extra-data.